### ISSUE TITLE
No Badge from completed Tutorial

### ISSUE THREAD

**Issue by marcobuescher** [SAPDocuments/Tutorials#3480]
**MissionURL**: https://developers.sap.com/mission.mobile-dev-kit-get-started.html

Get Started with the Mobile Development Kit

I get no Badget for that Mission while I made all Tutorials for that. :)

Can you have a look for that?

Thank you

Regards

**Comment by thecodester** [comment]
The badge takes about a week to be added to your profile -- a few more days perhaps since we just added it to our system. Please write back at the end of this week whether you received it.

**Comment by pfefferf** [comment]
Seems that this is an issue for the "mobile" tutorials/missions. I finished the mobile card tutorial (https://developers.sap.com/mission.mobile-cards-get-started.html) already weeks ago and the badge does not appear. But hey, it is just a badge ;)

**Comment by thecodester** [comment]
What day did you finish it?

**Comment by pfefferf** [comment]
On March 10, after issue #3188 was fixed.

**Comment by thecodester** [comment]
OK, I will check with the "badge people", who will arrive later on today. Will update you when I know something.

**Comment by marcobuescher** [comment]
No Badge from "Connect Microsoft Office 365 and SAP Data Through an SAPUI5 App ", too.

Regards


I just assume that it is a technical problem. Maybe it was just forgotten when researching?

**Comment by marcobuescher** [comment]
Hi,
I think I found the mistake. There is no badge. I completed the missions

- Connect Microsoft Office 365 and SAP Data Through an SAPUI5 App 
- Get Started with the Mobile Development Kit
- Develop Your First SAPUI5 Web App on Cloud Foundry

 and just assumed that there is a badge. However, the said missions are not listed on the overview page. "Currently Available Missions and Badges" (https://www.sap.com/community/resources/missions-badges.html). Maybe they will be introduced in retrospect ...

Regards...
